packages/pages/src/util/runtime.ts

```diff
@@ -32,8 +39,39 @@ class Runtime {
       throw new Error("Not running in Node.");
     }
 
+    if (!this.version) {
+      throw new Error("No version found.");
+    }
+
     return +this.version.split(".")[0];
   }
+
+  isBrowser(): boolean {
+    return (
+      typeof window != "undefined" &&
+      typeof window.document !== "undefined" &&
+      !("Deno" in window)
+    );
+  }
+
+  isDeno(): boolean {
+    return (
+      // @ts-expect-error Deno may be undefined
+      typeof Deno !== "undefined" &&
+      // @ts-expect-error Deno may be undefined
+      typeof Deno.version !== "undefined" &&
+      // @ts-expect-error Deno may be undefined
+      typeof Deno.version.deno !== "undefined"
+    );
+  }
+
+  isNode(): boolean {
```

Comment:
out of curiosity, why don't we use browser-or-node directly instead of writing it ourselves?

Reply:
just to limit dependencies we have it's so small

Reply:
browser-or-node seems pretty tiny though? https://github.com/flexdinesh/browser-or-node/blob/master/src/index.js

Reply:
I didn't know this existed when I made runtime. We could probably just use it. Though it's a breaking change to remove runtime completely, but we're in beta.

One thing ours has is that you can get the node version which I think is checked somewhere. I could be wrong and I had just added the version for fun in case we ever needed it. You'll have to check usages.

Reply:
We could definitely keep `getRuntime` and change the implementation to use browser-or-node. @tatimblin maybe we'd want to ask product what they would want here? I didn't realize this was a public export.

Reply:
I switched over to using the package, but since it exports variables instead of functions I was unable to mock different environments. Switched back to the calls I originally had.